README.md

```diff
@@ -44,13 +44,56 @@ Fetches a list of places according to the applied filter.
 
 ##### Arguments:
 - [`filter: Places.PlacesFilterJSON`](http://docs.sygictravelapi.com/js-sdk/master/interfaces/_places_filter_.placesfilterjson.html)
+
+###### PlacesFilter:
+
+```ts 
+// Filter full example
+const placeFilter: SygicTravelSDK.Places.PlacesFilterJSON = {
+    query: 'Tower', 
+    mapTiles: ['0121011'], // Matches places within map tile coordinate.  For details see [Bing Maps docs](https://msdn.microsoft.com/en-us/library/bb259689.aspx) or Maptiler.org
+    mapSpread: 1,
+    bounds: {
+        south: 50.123,
+        west: -0.55,
+        north: 50.523,
+        east: 0.05
+    },
+    categories: ['eating', 'sightseeing'], // filter out only certain categories
+    categoriesOperator: 'OR',
+    tags: ['indian_cousine', 'mexican_counsine'], 
+    tagsOperator: 'OR',
+    parents: ['city:1', 'city:5'], // filter out only places that has certain parents
+    parentsOperator: 'OR',
+    levels: ['poi'],
+    limit: 20,
+    zoom: 10
+}
+```
+
+| Property  | Description |
+| ---------- | ----------- |
+| query | Matches places containing the query within name, description ... |
+| mapTiles | Matches places within map tile coordinate.  For details see [Bing Maps docs](https://msdn.microsoft.com/en-us/library/bb259689.aspx) or [Maptiler.org](Maptiler.org) |
+| bounds | Limits results to area defined by bounds. The units are in degrees of latitude/logitude. |
+| mapSpread | Use mapSpread when you want to display the places on the map. The area is subdivided into more areas so that places cover map equally. Possible values are 0,1,2 or 3. This parameter requires bounds and zoom.|
```

Comment:
The area is subdivided into more areas so that places cover map equally => The area is subdivided into more areas so places cover map equally